jboss-eap-clustered-multivm-rhel8/README.md

```diff
@@ -47,12 +47,11 @@ Following is the Architecture:
 
 ![alt text](images/arch.png)
 
-To learn more about the JBoss Enterprise Application Platform, visit:
-https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/
+To learn more about the JBoss Enterprise Application Platform, visit [Red Hat EAP 7.2 documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/)
 
 ## Subscriptions and Costs - READ the instructions in their entirety before deploying!
 
-Additional documentation for deploying VM in Azure can be found here: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/
+[Linux virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/) provides additional documentation for deploying VMs in Azure.  
```

Comment:
We have completely changed the structure of the “Subscriptions and Costs” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback so that we can replicate the same format here.